_data/ambassadors.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Are we showing this info anywhere?

Reply:
No. The `Meet` component is currently hidden.